src/XMLSecurityDSig.php

```diff
@@ -107,9 +107,16 @@ class XMLSecurityDSig
     /**
      * @param string $prefix
      */
-    public function __construct($prefix='ds')
+    public function __construct($prefix='ds', $attributes=[])
     {
         $template = self::BASE_TEMPLATE;
+        if (count($attributes)) {
```

Comment:
@ahmadsyamim You can drop the if-statement.. It unnecessary.
If attributes is empty, the foreach will be skipped automatically

Reply:
@tvdijen 
Sorry for late reply. I have drop the if-statement.